### ISSUE TITLE
virus detected in some pre release binaries

### ISSUE THREAD

**Issue by mwuethrich** [git-for-windows/git#113]
My virus scanner does not like the binaries. Some examples:

Git-2.3.6.2-dev-preview-64-bit.exe: OK
- https://www.virustotal.com/en/file/5884b89412af357fb619a150cb2d82e150a924f8c1a53abfc0f7998917350395/analysis/1430053585/

PortableGit-2.3.6.2-dev-preview-64-bit.7z.exe: NOK
- https://www.virustotal.com/en/file/f835e8a15659baf1e656f737f9ff6e4dbc12df821f70c1fef619b57798e746c8/analysis/1430053100/

Git-2.3.5.8-dev-preview-64-bit.exe: NOK
- https://www.virustotal.com/en/file/3ed7e93f08e8f11922b99b7e5b18fb935729f715011b43c038a4d4fd6b3cf684/analysis/1430053913/

PortableGit-2.3.5.8-dev-preview-64-bit.7z.exe: NOK
https://www.virustotal.com/en/file/3ed7e93f08e8f11922b99b7e5b18fb935729f715011b43c038a4d4fd6b3cf684/analysis/1430053913/

Git 1.9.5 binaries downloaded from (https://msysgit.github.io/) are not affected.

**Comment by dscho** [comment]
Please contact your antivirus company. We use the same self extractor for years now, and everytime a virus scanner thought our installers was infected, we found out two things:

1. the virus scanner was wrong, and
2. the virus scanner companies were just happy to waste the time of non-customers.

So I fear that *you* will have to complain to them, I already lost too much life time to their work ethics.

**Comment by mwuethrich** [comment]
Actually, my virus scanner (McAfee) blocked it. The download worked fine, but during the installation the software blocked it. The online virus scanner results are just for show my issue.

**Comment by dscho** [comment]
You don't understand: since you are their customer, McAfee may act on your feedback. They won't bother with mine.

We have had *a lot* of incidences where antivirus programs declared our installers infected, and in 100% of those cases it turned out that the antivirus detection rang a false alarm.

I already spent more than enough time trying to resolve those issues in the past, and you take away even more of my time right now. The thing is: if I tell them that their software is broken, by virtue of not being their customer, they'll simply ignore me. I know. I tried.
